content/handbook/sales/process/version-check/index.md

```
@@ -0,0 +1,133 @@
+HTTP リファラーには、GitLab インスタンスのローカルまたはパブリックホスト名や IP アドレスが含まれる可能性があります。これは、管理者が GitLab Web インターフェースにどのようにアクセスしたかによって異なります。ローカルホスト名とローカル IP アドレスは、インスタンスが実行されているローカルネットワーク内でのみ関連性を持ち、到達可能です。したがって、ローカルホスト名は 'myownGitLab' のように何でも名前を付けることができます。パブリックホスト名や IP アドレスには、ホストネットワークの所有者に関する情報が含まれる可能性があります。例えば、HTTP リファラーに 'dev.gitlab.org' が含まれている場合、このインスタンスは GitLab が所有していると想定できます。
+
+## トラブルシューティング
+
+### 内部 API を使用してキャッシュをチェックする
+
+`/admin/version_check.json` エンドポイントにアクセスすることで、キャッシュデータがあるかどうか、また Rails を通じて UI に何が提供されているかを確認できます。
+
+レスポンスが `null` の場合、いくつかのステップを取ることができます:
+
+- ネットワークが `version.gitlab.com` への外部 API リクエストを許可／ホワイトリスト登録していることを確認します。
+- Sidekiq でバックグラウンドジョブがスケジュールされていて、何かの後ろで [スタックしていない](#ensure-sidekiq-jobs-are-in-healthy-state) ことを確認します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)